File: swagger/models/sale_payment.py
from .base_model import BaseModel


class SalePayment(BaseModel):
	"""
	no description available

	Attributes:

		id
		A unique identifier for this payment.

		amount
		The amount of this payment.

		method
		The method for this payment.

		type
		The type of payment.

		notes
		Notes about this payment.
	"""

	input_map = {
		'Id': 'id',
		'Amount': 'amount',
		'Method': 'method',
		'Type': 'type',
		'Notes': 'notes',
		}

	output_map = {
		'id': 'Id',
		'amount': 'Amount',
		'method': 'Method',
		'type': 'Type',
		'notes': 'Notes',
		}

	id = None
	amount = None
	method = None
	type = None
	notes = None

File: swagger/Models/SalePayment.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class SalePayment extends BaseModel
{
	/**
	 * A unique identifier for this payment.
	 * 
	 * @var integer format:int64
	 */
	public $id;
	/**
	 * The amount of this payment.
	 * 
	 * @var number format:double
	 */
	public $amount;
	/**
	 * The method for this payment.
	 * 
	 * @var integer format:int32
	 */
	public $method;
	/**
	 * The type of payment.
	 * 
	 * @var string 
	 */
	public $type;
	/**
	 * Notes about this payment.
	 * 
	 * @var string 
	 */
	public $notes;

	protected function getInputMap()
	{
		return [
			'Id' => ['id', 'integer', 'int64'],
			'Amount' => ['amount', 'number', 'double'],
			'Method' => ['method', 'integer', 'int32'],
			'Type' => ['type', 'string', null],
			'Notes' => ['notes', 'string', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'id' => ['Id', 'integer', 'int64'],
			'amount' => ['Amount', 'number', 'double'],
			'method' => ['Method', 'integer', 'int32'],
			'type' => ['Type', 'string', null],
			'notes' => ['Notes', 'string', null],
			];
	}
}
